circle.yml

```diff
@@ -1,17 +1,48 @@
+# This config uses parallelism. See https://circleci.com/docs/1.0/environment-variables/#parallelism
 machine:
   environment:
-    # Use the same port as Selenium server.
-    DRIVER_PORT: 4444
-    # Use the same base URL as Selenium server.
-    DRIVER_BASE_URL: wd/hub
+    browser_name: $(case $CIRCLE_NODE_INDEX in 0) browser_name='chrome' ;; 1) browser_name='firefox' ;; 2) browser_name='internet explorer' ;; 3) browser_name='android' ;; esac; echo $browser_name)
```

Comment:
Doing that is experimental: we're not sure there would be actual added value by hiding the complexity of a setup specifically done in CI in some other place than the CI config itself. Please timebox the experiment  :)